recipes/tcpcat/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMakeToolchain, CMake, CMakeDeps, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+import os
+
+
+class TcpcatConan(ConanFile):
+    name = "tcpcat"
+    package_type = "library"
+
+    # Optional metadata
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ydcpp/tcpcat"
+    description = "Simple C++ TCP Server and Client library."
+    topics = ("network", "tcp", "tcp server", "tcp client")
```

Comment:
```suggestion
    topics = ("network", "tcp", "tcp-server", "tcp-client")
```
Not 100% sure but I think spaces in topics are not allowed.

Reply:
updated, thanks.